### PR TITLE
check for server.jar before downloading

### PR DESCRIPTION
**Pull request by mflorida**
Check for existence of the Minecraft server.jar file to prevent redundant downloads.

### CODE REVIEW

**Comment by NBS111** [comment]
:100:

File: server-config.sh [hunk 1]
```diff
@@ -19,6 +19,8 @@ apt upgrade -y
 apt install git nginx openjdk-11-jdk-headless screen
 
 # install minecraft server
-curl https://launcher.mojang.com/v1/objects/1b557e7b033b583cd9f66746b7a9ab1ec1673ced/server.jar -o server.jar
+# download minecraft server if not already downloaded
+[[ ! -e server.jar ]] \
```
Comment (stephenlauck) [inline server-config.sh:23]: Could we just add an "if" here conditional?

Comment (NBS111) [reply server-config.sh:23]: I have no idea but sounds like a thing

Comment (stephenlauck) [reply server-config.sh:23]: It would make the `[[ stuff ]]` more explicit with `if [[ conditional statement ]]; then DO X; fi;`

Comment (thevinty) [reply server-config.sh]: :+1:

Comment (VTVL-CODE) [reply server-config.sh]: Thumbs Up.

**Comment by NBS111** [comment]
:+1:

File: server-config.sh (hunk 1 above)
Comment (VTVL-CODE) [reply server-config.sh]: 👍

**Comment by dabogsta** [comment]
:+1:

**Comment by VTVL-CODE** [comment]
👍

**Pull request merged**